src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/BaseContextMenuStrip.cs

```diff
@@ -152,7 +150,8 @@ private void RefreshSelectionMenuItem()
     private void AddVerbMenuItem()
     {
         //Add Designer Verbs..
-        IMenuCommandService menuCommandService = (IMenuCommandService)serviceProvider.GetService(typeof(IMenuCommandService));
+        IMenuCommandService? menuCommandService = serviceProvider.GetService<IMenuCommandService>();
+
         if (menuCommandService is not null)
```

Comment:
I believe we have a TryGetService method that will check for null so this can be simplified to
```suggestion
        if (serviceProvider.TryGetService(out IMenuCommandService? menuCommandService))
```